### PR TITLE
feat: Importable, generic handlers for getting and changing log levels

### PR DESCRIPTION
**Pull request by DRK3**
Signed-off-by: Derek Trider <Derek.Trider@securekey.com>

### CODE REVIEW

File: pkg/internal/support/httphandler_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	require.Equal(t, method, handler.Method())
+	require.NotNil(t, handler.Handle())
+
+	go handler.Handle()(nil, nil)
+
+	select {
+	case res := <-handled:
+		require.True(t, res)
+	case <-time.After(2 * time.Second):
+		t.Fatal("handler function didn't get executed")
```
Comment (Baha-sk) [inline pkg/internal/support/httphandler_test.go:37]: probably mention: `handler function timed out` instead

File: pkg/restapi/logspec/operation/operations.go [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+	err := json.NewDecoder(req.Body).Decode(&incomingLogSpec)
+	if err != nil {
+		commhttp.WriteErrorResponse(rw, http.StatusBadRequest, fmt.Sprintf(invalidLogSpec, err))
+		return
+	}
+
+	logLevelByModule := strings.Split(incomingLogSpec.Spec, ":")
+
+	defaultLogLevel := log.Level(-1)
```
Comment (Baha-sk) [inline pkg/restapi/logspec/operation/operations.go:106]: replace -1 with a const `INVALID_LEVEL` to be defined in `log` package

Comment (Baha-sk) [reply pkg/restapi/logspec/operation/operations.go:106]: there's no need to clutter the `log` package with this const, never mind my previous comment.

File: pkg/internal/support/httphandler_test.go (hunk 1 above)
Comment (DRK3) [reply pkg/internal/support/httphandler_test.go]: Done

**Pull request merged**